pom.xml

```diff
@@ -27,16 +27,13 @@
     <openapi-generator.version>5.3.1</openapi-generator.version>
     <mapstruct.version>1.4.1.Final</mapstruct.version>
     <json.version>20200518</json.version>
-    <hazelcast.version>4.0.2</hazelcast.version>
     <spring-kafka.version>2.8.2</spring-kafka.version>
     <wiremock.version>2.27.2</wiremock.version>
     <spring-kafka-test.version>2.8.2</spring-kafka-test.version>
```

Comment:
This line downgrades spring-kafka-test from 2.8.6 to 2.8.2 if a spring-kafke-test dependency gets added in future. Please remove this line.